.github/workflows/ci.yml

```diff
@@ -69,3 +69,36 @@ jobs:
         with:
           # disable gcov, not needed
           functionalities: gcov
+
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install doc dependencies
+        run: |
+          sudo apt update --yes && sudo apt install --yes git build-essential pandoc graphviz
+          pip install -U pip setuptools wheel
+          pip install -e .[docs]
```

Comment:
I think this is currently not working since there is no `docs` requirement in `setup.py`